### ISSUE TITLE
Spatial order ignored when displayed on OSX

### ISSUE THREAD

**Issue by Evizero** [JuliaImages/Images.jl#451]
Not sure what is going on, but I could swear this did not happen a few days ago. Basically I am using `permutedims` to test if my functions work correctly. However, it seems that either IJulia doesn't display the images correctly anymore, or I am imagining things.

Is the following normal behaviour ? I would have assumed that the images should be displayed upright in both situations, and I could swear this was the case a few days ago:

``` julia
using Images
using TestImages

img = testimage("lena_color_256")
```

![img1](https://cloud.githubusercontent.com/assets/10854026/13564997/b53dc088-e44d-11e5-83fc-bbaec7264248.png)

``` julia
show(img)
```

```
RGB Images.Image with:
  data: 256x256 Array{ColorTypes.RGB{FixedPointNumbers.UFixed{UInt8,8}},2}
  properties:
    imagedescription: <suppressed>
    spatialorder:  x y
    pixelspacing:  1 1
```

``` julia
img2 = permutedims(img, [2,1])
```

![img2](https://cloud.githubusercontent.com/assets/10854026/13565003/b9c450ae-e44d-11e5-852f-940d48667a41.png)

``` julia
show(img2)
```

```
RGB Images.Image with:
  data: 256x256 Array{ColorTypes.RGB{FixedPointNumbers.UFixed{UInt8,8}},2}
  properties:
    imagedescription: <suppressed>
    spatialorder:  y x
    pixelspacing:  1 1
```

Edit: Using `view`of `ImageView` works as I would expect it to (displayed upright in both cases)

Edit: I'm on the currently tagged versions for every package.

**Comment by timholy** [comment]
May be because of recent work I did in ImageMagick on EXIF, not sure.

Unfortunately, I didn't get to this before leaving on vacation, and I have a very low bandwidth connection. If this bothers you, take a look at the `writemime` method in Images and the `save(::stream)` method in whichever IO package (QuartzImageIO or ImageMagick) you're using.

**Comment by Evizero** [comment]
Now that I am using Tom's VisualRegressionTests this issue surfaced again. Basically on OSX if I do a `permutedims(testimg, [2, 1])` the image will be transposed as shown above. This doesn't seem to have anything to do with IJulia, but seems special to something on OSX (probably QuartzImageIO). Travis on OSX gets the same transposed result while while the linux one doesn't

**Comment by Evizero** [comment]
Here probably? https://github.com/JuliaIO/QuartzImageIO.jl/blob/master/src/QuartzImageIO.jl#L258
Unfortunately, I am not quite sure how that code really works there (or rather how it is supposed to work) or I would try to fix it right now. Maybe something like this: https://github.com/JuliaIO/ImageMagick.jl/blob/master/src/ImageMagick.jl#L176 has to happen?

**Comment by timholy** [comment]
Yep, that's gotta be it. Great detective work!

**Comment by timholy** [comment]
Closed by #577
